### PR TITLE
{lib}[gompi/2020b] fftlib v20170628

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+easyblock = 'MakeCp'
+
+name = 'fftlib'
+version = '20170628'
+_commit = '44c5fa0'
```
Comment (boegel) [inline easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb:5]: underscore-prefixed variables are indeed valid as local variables, but I think using `local_commit` instead is clearer?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3104.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/5df353c6fadcb6d1eac07e4011832519 for a full test report.

File: easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+homepage = 'https://github.com/flwende/fftlib'
+description = """A library that intercepts FFTW calls and adds features on top of it.
+ In particular, it enables FFT plan reuse when there are multiple calls for the same geometry."""
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+toolchainopts = {'openmp': True, 'cstd': 'c++11'}
+
+source_urls = ['https://github.com/flwende/fftlib/archive/']
+sources = [{'filename': '%s.zip' % _commit, 'extract_cmd': 'unzip %s'}]
```
Comment (boegel) [inline easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb:15]: Please rename on download:

```python
sources = [{'download_filename': '%s.zip' % local_commit, 'filename': SOURCE_ZIP}]
```

Why is the custom extract command required?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb (hunk 1 above)
Comment (migueldiascosta) [reply easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb:5]: same for `_options`?

File: easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb (hunk 2 above)
Comment (migueldiascosta) [reply easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb:15]: my bad (I had started with `CmdCp` easyblock and then realized `MakeCp` with a custom `build_cmd` was more appropriate)

File: easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb:5]: Yeah, using `local_*` consistently is better imho

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2624.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/3c45ece649cea8152c99083255e631c6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3502.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/c4d8ecf6782944879d20a43037824c00 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**